be/src/olap/tablet.cpp

```diff
@@ -839,7 +839,7 @@ void Tablet::_max_continuous_version_from_beginning_unlocked(Version* version, V
 }
 
 void Tablet::calculate_cumulative_point() {
-    WriteLock wrlock(&_meta_lock);
+    ReadLock wrlock(_meta_lock);
```

Comment:
Why readlock?

Reply:
sorry, it is my fault, I will fix it in next PR